src/Assets/Images/index.ts

```diff
@@ -8,5 +8,17 @@ import imgView3 from "./imgView3.png";
 import LandingImg from "./LandingImg.jpg";
 import RoomPicture1 from "./RoomPicture1.png";
 import RoomPicture2 from "./RoomPicture2.png";
+import RoomDetails1 from "./RoomDetails1.png";
+import RoomDetails2 from "./RoomDetails2.png";
+import RoomDetails3 from "./RoomDetails3.png";
+import ic_bedroom from "./ic_bedroom.png";
+import ic_ac from "./ic_ac 1.png";
+import ic_diningroom from "./ic_diningroom 1.png";
+import ic_kulkas from "./ic_kulkas.png";
+import ic_livingroom from "./ic_livingroom.png";
+import ic_wifi from "./ic_wifi 1.png";
```

Comment:
You can rename these icons to short and more meaningful names such as 
`import ic_wifi as wifi from "./ic_wifi 1.png";`